substra/sdk/backends/local/backend.py

```diff
@@ -95,7 +105,7 @@ def __add_compute_plan(
             ]
         ])
         compute_plan = models.ComputePlan(
-            compute_plan_id=uuid.uuid4().hex,
+            compute_plan_id="local_" + uuid.uuid4().hex,
```

Comment:
Could we define "local_" in a global constant? 

This would allow to easily change the constant value without forgetting an occurence.